Investigate the truncation effect (#34)
11michalis11/ED-EMS-game-paper: sections/queueing_model/truncation_effect/main.tex

\subsection{Truncation effect timings} \label{sec:truncation_effect}

The choice of the artificial parameters \(N\) and \(M\) is an important 
decision of the model.

> In the untruncated simulation these values are set to be infinite. 

Reviewer: ```suggestion
In the untruncated simulation these values are not needed.
```

> This is more computationally expensive to run.

Reviewer: ```suggestion
This is not possible when obtaining the steady state probabilities of the
finite state Markov chain.
```
